### ISSUE TITLE
Bug: Lang in DashboardArticleLink

### ISSUE THREAD

**Issue by flob38** [c2corg/c2c_ui#3404]
There is a problem with the lang when using this component in home page, because the lang isn't accessible in the rute params, so defaults to english.

_Originally posted by @brunobesson in https://github.com/c2corg/c2c_ui/pull/3335#discussion_r1149187976_

**Comment by flob38** [comment]
@brunobesson : there is the same issue with all types of documents (outing, route), no?

At this time, we display the title of the doc in the lang in which it was written.
Which comportement do you want ? Only display the documents written in the lang used in UI or in the user's preferences?

**Comment by brunobesson** [comment]
No, there is no problem in such cases because the URL contains the lang definition, which is retrieved. E.g. https://www.camptocamp.org/waypoints/1522843/fr/col-basei-
